### PR TITLE
Pull Request for branch mobile_finish

### PR DESCRIPTION
**Pull request by anita00001**
In this branch, we have created the middle section of the portfolio page.

- The middle section mainly contains two sections: 1. class="works" and 2. class="about_me".
1. section class="works" have 3 sub-sections, namely class="title", class="feature" and class="frame"
2. section class="about_me" have 3 sub-sections, namely class="description", class="resume" and class="competence"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
         </ul>
         </nav>
         <a class="bg_img"></a>
-  </footer>
-  </section>
+      </article>
+    </section>
+
+    <section class="works">
+      <section class="title">My Recent Works</section>
```
Comment (bobb-Rob) [inline index.html]: - [x] The works section header should use `hx` i.e `h2`, `h3` etc. Kindly replace this section tag with any of the hx. This sis essentially going to make your HTML structure semantically correct and improve the page SEO.

File: index.html [hunk 2]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
...
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+      </section>
+
+      <section class="frame">
+        <section class="card card1">
+          <h1>Professional Art<br>Printing data</h1>
```
Comment (bobb-Rob) [inline index.html]: - [x] To help improve your page SEO, it is best practice to have only one `h1` tag on an HTML page. Kindly change all the `h1` tags in each project card to any other `hx`, could be `h2`.

File: index.html [hunk 3]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
...
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card2">
+          <h1>Professional Art<br>Printing data</h1>
```
Comment (bobb-Rob) [inline index.html]: - [x] To help improve your page SEO, it is best practice to have only one h1 tag on an HTML page. Kindly change all the h1 tags in each project card to any other hx, could be h2.

File: index.html [hunk 4]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
...
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card3">
+          <h1>Professional Art<br>Printing data</h1>
```
Comment (bobb-Rob) [inline index.html]: - [x] To help improve your page SEO, it is best practice to have only one h1 tag on an HTML page. Kindly change all the h1 tags in each project card to any other hx, could be h2.

File: index.html [hunk 5]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
...
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+      </section>
+    </section>
+
+    <section class="about_me">
+      <section class="description">
+        <h1>About me</h1>
```
Comment (bobb-Rob) [inline index.html]: - [x] To help improve your page SEO, it is best practice to have only one h1 tag on an HTML page. Kindly change all the h1 tags in each project card to any other hx, could be h2.

File: index.html [hunk 6]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
...
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </section>
+      <section class="resume">
+        <button>Get My Resume</button>
+        <a class="about_bgimg"></a>
```
Comment (bobb-Rob) [inline index.html]: - [x] Background images are best on the page using CSS property `background-image`  and not with an element as you have use here. Kindly remove this `<a class="about_bgimg"></a>`.

File: index.html [hunk 7]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
...
+      </section>
+    </section>
+
+    <section class="about_me">
+      <section class="description">
+        <h1>About me</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </section>
+      <section class="resume">
+        <button>Get My Resume</button>
```
Comment (bobb-Rob) [inline index.html]: - [x] Please ensure your button looks exactly like the one on the Figma design. Please implement all the button state as shown in the third diagram bellow, 

**Your button**
image

**Figma button**
image

**Button States**
image

File: index.html [hunk 8]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
...
+        </ul>
+        <ul>
+          <li><div class="lang"><img src="images_2/circle.png">Skills</div>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </li>
+        </ul>
```
Comment (bobb-Rob) [inline index.html]: - [x] In This section, the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.

File: index.html [hunk 9]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
...
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+      </section>
+    </section>
+
+    <section class="about_me">
```
Comment (bobb-Rob) [inline index.html:127]: - [x] This section is expected to use flexbox to align its content. I see you are using flexbox already, but the section element above should flex declared, with `flex-direction: column`. Kindly implement this.

File: index.html [hunk 10]
```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
...
+        </section>
+        <section class="card card6">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
```
Comment (bobb-Rob) [inline index.html]: - [x] Please also, implement the hover and active state on this button.
- [x] Every button element should have a type property on it. Please fix this.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @anita00001 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by huseyinbiyik — APPROVED** [review]
Hi @anita00001,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights

## Optional suggestions
[OPTIONAL] Good job with the implementation of this section but it is not a part of the project. You can remove it 👨‍🎨
Screenshot 2023-01-13 at 8 24 06 PM


[OPTIONAL] Your implementation looks great but there is a small white space here on the right. Kindly fix that.
Screenshot 2023-01-13 at 8 25 30 PM


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**